### PR TITLE
Moves deterministic encoded CBOR language back to CBOR section 

### PR DESCRIPTION
**Pull request by jonnycrunch**
This PR undoes the `editorial` changes to get us back on having the discussion regarding Deterministically Encoded CBOR back to the conversations.  

The language builds on the updated CBOR spec [RFC #8949](https://tools.ietf.org/html/rfc8949#section-4.2)

***
<a href="https://pr-preview.s3.amazonaws.com/jonnycrunch/did-core/pull/586.html" title="Last updated on Jan 28, 2021, 5:12 PM UTC (38e9f0e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/586/64397a7...jonnycrunch:38e9f0e.html" title="Last updated on Jan 28, 2021, 5:12 PM UTC (38e9f0e)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3097,6 +3097,20 @@ <h3>Production</h3>
           </li>
         </ul>
 
+        <p>
+          To produce a deterministic canonical CBOR representation of a DID document and faciliate maximal lossless compatiblity with other 
+          core representations via the Abstract Data Model the following constraints of a CBOR representation of a DID Document model MUST be followed:
+
```
Comment (TallTed) [inline index.html:3103]: typos and line lengths
```suggestion
To produce a deterministic canonical CBOR representation of a DID
document and facilitate maximal lossless compatibility with other core
representations via the Abstract Data Model the following constraints
of a CBOR representation of a DID Document model MUST be followed:
        </p>
```

File: index.html [hunk 2]
```diff
@@ -3097,6 +3097,20 @@ <h3>Production</h3>
...
+          core representations via the Abstract Data Model the following constraints of a CBOR representation of a DID Document model MUST be followed:
+          
+          <ul>
+            <li>Property names MUST be represented as text string (major type 3) and contain only UTF-8 strings. </li>
+            <li>Undefined Values of Required Properties as defined in <a href="#data-model">the Data Model</a> that are absent from the CBOR representation SHOULD be labeled with Primitive type (major type 7) with value 23 (Undefined value).  </li>
+            <li>Property names in each CBOR map MUST be unique. </li>
+            <li>Integer encoding MUST be as short as possible.</li>
+            <li>The expression of lengths in CBOR major types 2 through 5 MUST be as short as possible. </li>
+            <li>The keys in every map must be sorted lowest value to highest. Sorting is performed on the bytes of the representation of the keys. If two keys have different lengths, the shorter one sorts earlier.  If two keys have the same length, the one with the lower value in (byte-wise) lexical order sorts earlier. </li>
```
Comment (TallTed) [inline index.html:3110]:
```suggestion
            <li>
Property names MUST be represented as text string (major type 3) and
contain only UTF-8 strings.
            </li>
            <li>
Undefined Values of Required Properties as defined in <a
href="#data-model">the Data Model</a> that are absent from the CBOR
representation SHOULD be labeled with Primitive type (major type 7)
with value 23 (Undefined value).
            </li>
            <li>
Property names in each CBOR map MUST be unique.
            </li>
            <li>
Integer encoding MUST be as short as possible.
            </li>
            <li>
The expression of lengths in CBOR major types 2 through 5 MUST be as
short as possible.
            </li>
            <li>
The keys in every map must be sorted lowest value to highest. Sorting
is performed on the bytes of the representation of the keys. If two
keys have different lengths, the shorter one sorts earlier.  If two
keys have the same length, the one with the lower value in (byte-wise)
lexical order sorts earlier.
            </li>
```
line lengths

**Comment by iherman** [comment]
@jonnycrunch did you use a different github persona than the one you registered with the WG? Otherwise I don't know why this PR wasn't accepted.

I will set this as editorial, which will release the lock, but you should check, please. Thx

**Comment by msporny** [comment]
@jonnycrunch, I'm confused by this PR. The text was already moved back in this commit: https://github.com/w3c/did-core/commit/5b7c489e3231a3e31171c7f40c71973c197a612f which was announced to the WG here: https://lists.w3.org/Archives/Member/member-did-wg/2021Jan/0002.html

Is there some difference between the text in the spec and what you're proposing in the PR?

**Comment by brentzundel** [comment]
@jonnycrunch I believe the changes you are recommending here have already been made.

**Review by OR13 — CHANGES_REQUESTED** [review]
make no mention on canonical, based on the comments from the call... prefer to not see that language / think this should be closed.

**Comment by msporny** [comment]
@brentzundel wrote:
> @jonnycrunch I believe the changes you are recommending here have already been made.

I checked a few days ago and checked again today per @jricher's request -- the changes in this PR were merged into the specification before this PR was raised. There are wording differences due to a refactoring of language that @peacekeeper applied to the spec wrt. ADM entry keys, etc. This PR undoes some of those changes. The group also discussed this PR during the special topic call today and decided to close it.

For those reasons, this PR is marked pending close and will be closed in 7 days or when @jonnycrunch confirms that we can close this PR, whichever event happens sooner.

**Comment by msporny** [comment]
> This PR is marked pending close and will be closed in 7 days or when @jonnycrunch confirms that we can close this PR, whichever event happens sooner.

7 days have elapsed, closing.

**Pull request closed without merging**